src/App.js

```diff
@@ -33,11 +37,33 @@ export default function App() {
   }, []);
 
   return (
-    <Main>
+    <>
```

Comment:
Unfortunately, I cannot comment one line below. As far as I can see you online have an `h1` element in your `header`. A header is a sectioning element and is only needed if you wanna group multiple elements. As you only have one element here, you do not need the header. Instead you can just use only the `h1` and style it accordingly.

Reply:
Thank you for the hint. The header section becomes in the next user stories a kind of a map legend with multiple elements in it. So the `h1` element won't stay alone much longer